frontend/cypress/tests/codeView/codeView_hideFileDetails.js

```diff
@@ -0,0 +1,103 @@
+describe('hide all file details', () => {
```

Comment:
not sure if we need tests on hiding file details as it is not a crucial feature, the test might be taking up time instead

Reply:
@fzdy1914 ur opinion on this?

Reply:
I am OK with this testing actually. How much time will this extra test need?

Reply:
Together both the  "hide" tests take a total of approximately 5000 ms, so 5 seconds

Reply:
The time needed is OK for me.